.github/workflows/docsearch.yml

```diff
@@ -0,0 +1,20 @@
+name: Run algolia dockersearch
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        echo "APPLICATION_ID=$APPLICATION_ID" >> .env
+        echo "API_KEY=$API_KEY" >> .env
+        chmod +x ./bin/crawl.sh
+        ./bin/crawl.sh
+      env:
```

Comment:
I think you should put env above run?

Similar to this:
![image](https://user-images.githubusercontent.com/6205961/155690171-b94c6a5d-f288-479a-a715-fdc9acd5ae2a.png)

Reply:
It doesn't matter in `yml`.
It's all `key: value`

Reply:
yeah, but the syntax highlighting doesn't make it clear.